scripts/ioc_deploy.py

```diff
@@ -1071,29 +1144,50 @@ def rearrange_sys_argv_for_subcommands():
 
     Otherwise, the first example here is interpretted as if -p was never passed,
     which could be confusing.
+
```

Comment:
interpretted -> interpreted above, but I can't comment on it since there was no diff up there.

I think it's probably good to leave a really clear note that this mutates sys.argv in place.  As a function that isn't passed any arguments, I'd probably not expect any side effects past the return value.  

The name of the function is clear, but I just want to make sure we're extra clear here

Reply:
I'll do these